Languages/Dutch/Keyed/Multiplayer.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<LanguageData>
+
+  <MpSimulating>Simuleren</MpSimulating>
+  <MpSimulatingServer>Simuleren van de server</MpSimulatingServer>
+  <MpLoading>Bezig met opslaan</MpLoading>
+  <MpSaving>Aan het opslaan</MpSaving>
+  
+  <MpInvalidPort>Ongeldige poort</MpInvalidPort>
+  <MpInvalidAddress>Ongeldig adres</MpInvalidAddress>
+  <MpInvalidGameName>Ongeldige spelnaam</MpInvalidGameName>
+  
+  <MpWrongProtocol>Verkeerde protocolversie</MpWrongProtocol>
+  <MpWrongMultiplayerVersionInfo>De server wordt gehost op Multiplayer {0} (protocol {1}).</MpWrongMultiplayerVersionInfo>
+  
+  <MpWrongDefs>Modeldefinitie komt niet overeen</MpWrongDefs>
+  <MpWrongDefsInfo>Zorg ervoor dat uw geïnstalleerde mods, hun versie en hun volgorde overeen komen met die van de host.</MpWrongDefsInfo>
+  <MpModList>Wijzigingslijst</MpModList>
+  <MpSeeModList>Zie wijzigingslijst</MpSeeModList>
+  
+  <MpServerClosed>Server gesloten</MpServerClosed>
+  <MpServerFull>De server is vol</MpServerFull>
+  <MpKicked>Je bent gekickt van de server</MpKicked>
+
+  <MpInvalidUsernameAlreadyPlaying>Gebruiker is al aanwezig in de game</MpInvalidUsernameAlreadyPlaying>
+  <MpInvalidUsernameChars>Gebruikersnaam bevat ongeldige tekens</MpInvalidUsernameChars>
+  <MpInvalidUsernameLength>Gebruikersnaam heeft een ongeldige lengte</MpInvalidUsernameLength>
+  <MpChangeUsernameInfo>U kunt uw gebruikersnaam wijzigen in de wijzigingsinstellingen.</MpChangeUsernameInfo>
+  
+  <MpPlayerConnected>{0} heeft zich aangesloten</MpPlayerConnected>
+  <MpPlayerDisconnected>{0} is vertrokken</MpPlayerDisconnected>
+  
+  <MpReplayOutdated>(Outdated)</MpReplayOutdated>
+  <MpReplayOutdatedDesc1>Deze herhaling is gemaakt voor de protocolversie {0}, maar je bent bezig met de versie die je gebruikt... {1}.</MpReplayOutdatedDesc1>
+  <MpReplayOutdatedDesc2>De herhalingen worden mogelijk niet correct afgespeeld.</MpReplayOutdatedDesc2>
+  <MpReplayOutdatedDesc3>Mogelijke incompatibiliteiten worden vermeld in het releaseverleden.</MpReplayOutdatedDesc3>
+  <MpReplayOutdatedDesc4>Dit bestand bevat aanvullende informatie en de hosting ervan zou goed moeten werken.</MpReplayOutdatedDesc4>

```

Comment:
What is the English translation for this?

Reply:
This file contains additional information and hosting from it *should work fine.*